client/src/pages/customers/form.tsx

```
@@ -0,0 +1,236 @@
+}
+
+export function CustomerCardForm({
+  form,
+  isCreate = false,
+}: CustomerCardFormProps) {
+  return (
+    <ScrollArea className="h-[500px]">
+      <div className="space-y-4 mr-4 ml-4 pb-4">
+        {isCreate && (
+          <form.Field
+            children={(field) => {
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)